modules/nf-core/bracken/build/main.nf

```diff
@@ -0,0 +1,47 @@
+process BRACKEN_BUILD {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bracken:2.9--py38h2494328_0':
+        'biocontainers/bracken:2.9--py38h2494328_0' }"
+
+    input:
+    tuple val(meta), path(kraken2db)
+
```

Comment:
Do you think the read length parameter should also be included as input since it affect the output file names?

Reply:
There is a default (100bp) so I otherwise leave that to the pipeline developer to inject that into `args` :), and the current output glob doesn't explicitly pick them up so shouldn't matter